00-programming-fundamentals/intro-to-automated-tests.md

```diff
@@ -124,91 +124,108 @@ We are now setup, but we haven't actually written a test yet.
 
 ### A Tale of Two Styles
 
-In the TDD World there are two styles of testing.  In the first more traditional method people use *assertions* which are statements that check if a value is what it should be.  The other method is a subset of TDD, called Behavior-Driven Development (BDD) which accomplishes the same thing in a more English-friendly fashion that even non-developers can understand. 
+In the TDD World there are two styles of testing.  In the first more traditional method people use *assertions* which are statements that check if a value is what it should be.  The other method is a subset of TDD, called Behavior-Driven Development (BDD) which accomplishes the same thing in a more English-friendly fashion that even non-developers can understand.  At Ada we will use the second BDD style of testing.  You should know assertion-style testing is a thing, and that it accomplishes the same job as our behavior-driven development, but we will not require you to write assertion-style tests.  
```

Comment:
I know that BDD was literally written intended to help non-developers understand the code, but is there a way that we can word it so it doesn't use the phrase "that even non-developers can understand"?

Like "business analysts" or something!

I want to transition away from language that reinforces that we're choosing to do something because they are _non-developers_ , because they are/are becoming developers! Also, "technical vs. non-technical" is a weird whole discussion